src/app/settings/shelf/action-panel/action-panel-list/action-panel-list.component.html

```diff
@@ -0,0 +1,90 @@
+<div class="c-action-panel-list">
```

Comment:
This components does a lof of the stuff that is supposed to be encapsulated by a reusable TableComponent.
Instead of making a reuseable component, you have made global CSS classes. We don't want global CSS classes, we want reusable components.

If we need to use another table somewhere else in the system, we need to reimplemt a lot of the markup/logic for a table. That goes against the point of reusable components.